rest_framework_json_api/utils.py

```diff
@@ -304,6 +304,21 @@ def get_resource_type_from_serializer(serializer):
     )
 
 
+def get_resource_id_from_instance(resource_instance, resource):
```

Comment:
I would rename it now as it is not retrieving more than just an instance to `get_resource_id`.